ocs_ci/templates/app-pods/esclient.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: es-dumper
+  namespace: elastic-system
+spec:
+  containers:
+   - name: esdumpper
+     image: quay.io/avili/esdump:v0.1
```

Comment:
we can move this image to ocsci

Reply:
@vasukulkarni There are chances that we may use this container image in customer systems for debugging. I'm not sure if there is any security compliance needed. Do you think of any implications? (I may be overthinking, but just want to double-check)

Reply:
moved

Reply:
@keesturam I think Avi's image is originally from docker, quay gives the security scan, we should look into the security scan and probably update the image if anything serious shows up.